src/components/shareModal/ShareModal.types.ts

```diff
@@ -14,6 +14,8 @@ export interface ModalBodyProps {
   showCopyToClipboard: boolean;
   windowLocationHref: string;
   onClickCopyToClipboard(showToast: boolean): void;
+  showShareToFeed: boolean;
```

Comment:
Not required if you set the default value

Reply:
Done.